python/pyspark/pandas/spark/utils.py

```diff
@@ -143,9 +150,15 @@ def force_decimal_precision_scale(
     ...             StructField('b', StringType(), True)])),
     ...     StructField("B", DecimalType(30, 15), False)]),
     ...     precision=30, scale=15)  # doctest: +NORMALIZE_WHITESPACE
-    StructType(List(StructField(A,StructType(List(StructField(a,MapType(DecimalType(30,15),\
-ArrayType(DecimalType(30,15),false),false),false),StructField(b,StringType,true))),true),\
-StructField(B,DecimalType(30,15),false)))
+    StructType([StructField('A',
+                            StructType([StructField('a',
```

Comment:
Cab we get rid of all these white spaces? or at least two space indentation.

Reply:
I'll take another look. IIRC there was some nuance with doctests and wrapped whitespace that was difficult to work around, which is why it has the odd hanging indents here.